pdns/stubresolver.cc

```diff
@@ -14,9 +16,15 @@
 #include "statbag.hh"
 #include "stubresolver.hh"
 
+#define LOCAL_RESOLV_CONF_PATH "/etc/resolv.conf"
+
 // s_resolversForStub contains the ComboAddresses that are used by
 // stubDoResolve
 static vector<ComboAddress> s_resolversForStub;
+static pthread_mutex_t s_resolversForStubLock = PTHREAD_MUTEX_INITIALIZER;
+
+// /etc/resolv.conf last modification time
+static auto s_localResolvConfMtime = 0;
```

Comment:
In this case, you leave the compiler guessing with = 0. I suggest static time_t s_localResolvConfTime